src/components/Main.js

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import Nav from './Nav';
+import AccordianDemo from '../demos/AccordianDemo';
```

Comment:
Spelling - should be "Accordion". Occurs in several places

Reply:
Fixed